hooks/openfeature-hooks-opentelemetry/pyproject.toml

```diff
@@ -27,17 +27,23 @@ Homepage = "https://github.com/open-feature/python-sdk-contrib"
 
```

Comment:
I'm going to revert the changes in this file and put them into main directly to prevent changelog entries for the hook that aren't really relevant.